src/cgame/rocket/rocket.h

```diff
@@ -88,6 +88,10 @@ struct UnvPropertyId {
 	static Rml::Core::PropertyId Orientation;
 };
 
+#define REGISTER_ELEMENT( tag, clazz ) static std::unique_ptr<Rml::Core::ElementInstancerGeneric< clazz >> instancer_##clazz; \
```

Comment:
I was not very fond of seeing this macro in the C++ side, but in C++ was okayish considering the scope of the change (and how much it's desired).
Now it lands in a header and is meant to be used by other stuff, so I'd rather see a template function.

Attached patch *compiles* but I have not checked that it *works* (use with care!).
[0001-suggested-patch.patch.txt](https://github.com/Unvanquished/Unvanquished/files/8078310/0001-suggested-patch.patch.txt)

Reply:
Done.